app/controllers/application_controller.rb

```diff
@@ -2,4 +2,5 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  include SessionsHelper
```

Comment:
In rails helpers are normally only for the view (unlike sinatra.) The methods in sessions_controller may as well be instance methods in ApplicationController. If any need to be accessible from views also, declare them with helper_method in the controller.